src/install.js

```diff
@@ -1,28 +1,40 @@
 import * as components from './components';
 import {
+  DatePicker,
   VueCtkDateTimePicker,
   VueNumberInput,
   VueTelInput,
 } from './packages';
 
 import 'vue-tel-input/dist/vue-tel-input.css';
 
-const install = (Vue, {prefix} = {}) => {
+const $omitDeep = require('omit-deep');
+
+const install = (app, {prefix, loadComponents = true} = {}) => {
   for (let key in components) {
```

Comment:
why did we not replace this code with conditional logic?

Reply:
Ahh. I put the conditional logic in right after that. But I didn't look closely enough to realize that code was doing the same thing as the conditional logic. I'll double check the other packages I edited with the conditional logic.